README.md

```diff
@@ -246,6 +248,76 @@ expect(wrapper.find('#child')).to.have.descendants('#last')
 expect(wrapper).to.not.have.descendants('#root1')
 ```
 
+#### `descendantsCount(selector)`
+
+| render | mount | shallow |
+| -------|-------|-------- |
+| yes    | yes   | yes     |
+
+
+Assert that the wrapper contains a exact amount of descendants matching the given selector.
+Defaults to looking for exactly 1 descendent
+
+```js
+import React from 'react'
+import {mount, render, shallow} from 'enzyme'
+
+class Fixture extends React.Component {
+  render () {
+    return (
+      <div id='root'>
+        <span id='child'>
+          <span id='last'></span>
+          <span id='last'></span>
```

Comment:
Not sure what the status of this PR is, but just as an FYI if you're adding examples make sure it adheres to best practices, in this case don't use the same `id` more than once.